PopPUNK/network.py

```diff
@@ -281,13 +282,19 @@ def constructNetwork(rlist, qlist, assignments, within_label,
             from :func:`~PopPUNK.bgmm.findWithinLabel`
         summarise (bool)
             Whether to calculate and print network summaries with :func:`~networkSummary`
-
             (default = True)
         edge_list (bool)
             Whether input is edges, tuples of (v1, v2). Used with lineage assignment
         weights (numpy.array)
             If passed, the core,accessory distances for each assignment, which will
             be annotated as an edge attribute
+        weights_type (str)
+            Specifies the type of weight to be annotated on the graph - core, accessory
+            or Euclidean distance
+        sparse (bool)
+            Whether input is a sparse distance matrix
```

Comment:
Should be able to infer this from the type of the input?

Reply:
@johnlees Do you mean inferring from file name, or is there a smarter method?

Reply:
You should be able to use something like:
```
isinstance(sparse_input, scipy.sparse.coo_matrix)
```
Have a look with `type(sparse_input)` to double check what the comparison should be

Reply:
Great, thanks!